core/src/avm2/globals/flash/net/NetConnection.as

```diff
@@ -3,25 +3,42 @@ package flash.net {
     import flash.errors.IOError;
     import __ruffle__.stub_method;
 
+[Ruffle(InstanceAllocator)]
     public class NetConnection extends EventDispatcher {
 
         public static var defaultObjectEncoding:uint = 3;
 
         public var objectEncoding:uint = NetConnection.defaultObjectEncoding;
+        public var client:Object = this;
+        public var maxPeerConnections:uint = 8;
+        public var proxyType:String = "none";
```

Comment:
Aren't these getters/setters in Flash?

Reply:
Yes but they don't need to be, they don't do anything without rtmp so might as well not be :D